recipes-browser/wpewebkit/wpewebkit_2.22.2.bb

```diff
@@ -2,11 +2,10 @@ require wpewebkit.inc
 
 FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
 SRC_URI = "https://wpewebkit.org/releases/${PN}-${PV}.tar.xz \
-           file://0001-REGRESSION-r217927-WPE-2.22-GSTREAMER_GL-Video-appea.patch \
```

Comment:
Can this file be removed too?

Reply:
Test build in: https://gitlab.com/saavedra.pablo/meta-perf-browser/commit/679d597b2df0ed811db34384f88e16822f57b033

Reply:
> Can this file be removed too?

No, indeed. Thanks.